readthedocs/donate/signals.py

```diff
@@ -165,13 +165,14 @@ def attach_promo_data(sender, **kwargs):
         show_promo = False
 
     if PROMO_GEO_PATH:
+        from geoip2.errors import AddressNotFoundError  # noqa
```

Comment:
Does it make sense to move this to a top level conditional import? I'm not sure of the performance hit here.